examples/opensora_hpcai/opensora/datasets/video_dataset_refactored.py

```diff
@@ -32,6 +34,22 @@ def create_infer_transforms(target_size: Tuple[int, int], interpolation=Inter.BI
     )
 
 
+class BucketResizeCrop:
+    def __init__(self, buckets: Bucket):
+        self._transforms = {}  # is this reasonable? There are 350+ buckets
+        for name, lengths in buckets.ar_criteria.items():
+            self._transforms[name] = {}
+            for length, ars in lengths.items():
+                self._transforms[name][str(length)] = {}
+                for ar, hw in ars.items():
+                    self._transforms[name][str(length)][ar] = Compose(
+                        [Resize(min(hw), interpolation=Inter.BILINEAR), CenterCrop(hw)]
```

Comment:
Need to improve the transformation considering different AR.

src: 1024, 1024
tar: (576, 1024)

current:
resize: (576, 576) 
center crop:  (576, 1024)   black region will be filled on left and right side